jazzmin/templatetags/jazzmin.py

```diff
@@ -192,5 +192,4 @@ def debug(value):
     """
     Add in a breakpoint here and use filter in templates for debugging ;)
     """
-    import ipdb; ipdb.set_trace()
```

Comment:
I think this is exactly what this function is for.. if you remove the BP, won't do what is for? Anyway, I added a #NOQA tag in my branch to this, to prevent flakes.

Reply:
lol it is what its for, but im removing it as leaving it in would mean we need to add ipython and ipdb with the package

Reply:
oh gotcha! 👍 I guess we can get rid of the function then